pages/4_Documentation.py

```diff
@@ -510,3 +510,13 @@
 whatever classification makes sense for the type of evidence. One or more columns
 with a similar format can be specified for the same protein.""")
 
+st.subheader('PFAM module')
+
+st.write("""This module uses results retrieved from PFAM, reporting the identified
+         domains for the given protein and the respective residue ranges for each domain.
+         This module produces the following columns:""")
+
+data = [ ( "PFAM_domain_annotations",
+           "PFAM domains associated with the given residue in the protein",
+           "single string reporting the PFAM domain code") ]
+st.table(pd.DataFrame(data, columns=['Column', 'Description', 'Possible values']))
```

Comment:
to be upgraded due to current changes

a nitpick: I think the official wording of the database is `Pfam`